### ISSUE TITLE
Model taking too long to predict

### ISSUE THREAD

**Issue by ud2195** [ThilinaRajapakse/simpletransformers#1460]
I finetuned a bert model for a binary classification task, model can be found here - https://huggingface.co/emilyalsentzer/Bio_ClinicalBERT

everything works fine what is bothering me is the inference time for it to predict on CPU.
This is how i am doing the prediction

```
from simpletransformers.classification import ClassificationModel
model = ClassificationModel("bert", "/Users/xxxxx/ade_clf/", use_cuda=False)
model.predict(['Vincristine induced cranial polyneuropathy.'])

```
The `PREDICT` operation takes ~3.5 seconds. 
Can i somehow bring it down ?

**Comment by paulommaia** [comment]
You need to use the predict function from the model, not the wrapper. Let me know if those code helps you, I used it for a similar issue
`
def predict_class(model, tokenizer, query):

    class_list = model.args.labels_list
    sent_tk = tokenizer(query.lower(), return_tensors="pt")
    outputs = model.model(**sent_tk)

    output_numpy = softmax(outputs[0].detach().numpy().ravel())
    argmax_idx = np.argmax(output_numpy)

    predicted_class = class_list[argmax_idx]

    print("Predicted class:", predicted_class)

    score = float(output_numpy[argmax_idx])

    return {"predicted_class": predicted_class, "score": score}`

**Comment by sieuchuoicb** [comment]
I have the same problem. I find out that the predict function always take 3 - 4 seconds for the first time predict, and 0.1 seconds for the rest after the first time.
The easiest way to solve it, is writing an API (flask, fastapi), that's what i'm doing now.
